.travis.yml

```
@@ -28,7 +28,7 @@ before_install:
   - travis_retry wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
   - >
     RETRIES=0;
-    until (timeout 30 bash miniconda.sh -b -p $HOME/miniconda) || [[ $RETRIES -eq 4 ]]; do
+    until (timeout 180 bash miniconda.sh -b -p $HOME/miniconda) || [[ $RETRIES -eq 4 ]]; do
```

also kill hard if we reach the timeout
```suggestion
    until (timeout --signal=KILL 180 bash miniconda.sh -b -p $HOME/miniconda) || [[ $RETRIES -eq 4 ]]; do
```